python/llm/example/CPU/HF-Transformers-AutoModels/Model/flan-t5/README.md

```diff
@@ -0,0 +1,69 @@
+# Flan-t5
+
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on flan-t5 models. For illustration purposes, we utilize the [google/flan-t5-xxl](https://huggingface.co/google/flan-t5-xxl) as a reference flan-t5 model.
+
+## 0. Requirements
+To run these examples with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a flan-t5 model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install bigdl-llm[all] # install bigdl-llm with 'all' option
+```
+
+### 2. Run
+```
+python ./generate.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --prompt PROMPT --n-predict N_PREDICT
```

Comment:
I feel it maybe better to provide command line `python ./generate.py --prompt 'What is AI?'` which can be directly used. Please refer to https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/mistral/README.md to update section2-Run.